### ISSUE TITLE
Replace "add stage" in view bar with more familiar term

### ISSUE THREAD

**Issue by brimoor** [voxel51/fiftyone#477]
From @briankelly: “Stage” feels like non-standard lingo to me. I understand what it’s getting at now (Oh, I can do multiple transforms.. in a pipeline), but this wasn’t as obvious to me as I wanted it to be

![image (1)](https://user-images.githubusercontent.com/25985824/91582062-4c33ad80-e91d-11ea-9b05-6e7dcf7eb715.png)

**Comment by brimoor** [comment]
I agree on this. I think the best solution is just to follow Claire's suggestion to replace `+ add stage` with `+ search samples` as the placeholder text for the empty view bar state.

This doesn't mean we need to rename "stage" throughout the entire app/codebase or anything like that. In fact, I believe that `+ add stage` is the only occurrence of the word "stage" in the App anyway. 

The motivation for `+ search samples` would be to make it more clear what **outcome** you would be achieving by engaging with this bar: 

<img width="961" alt="Screen Shot 2020-08-28 at 10 52 15 AM" src="https://user-images.githubusercontent.com/25985824/91582301-9d43a180-e91d-11ea-8ec5-c75c40a72bcb.png">

**Comment by benjaminpkane** [comment]
I disagree here. I am happy to change this if it is what the majority feels. So I think it would be good to hear from other team members. I think what is decided here will inform a lot of other decisions.

My view is that FiftyOne is a technical tool, and therefore the technical language should be used.
